BUILD

```diff
@@ -120,15 +120,11 @@ PYTHON3.update({
 })
 
 NODEJS = {
-    "{REGISTRY}/{PROJECT_ID}/nodejs:12": "//nodejs:nodejs12_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:14": "//nodejs:nodejs14_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs:12-debug": "//nodejs:nodejs12_debug_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:latest": "//nodejs:nodejs16_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:debug": "//nodejs:nodejs16_debug_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:12": "//nodejs:nodejs12_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:14": "//nodejs:nodejs14_amd64_debian11",
-    "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:12-debug": "//nodejs:nodejs12_debug_amd64_debian11",
     "{REGISTRY}/{PROJECT_ID}/nodejs-debian11:14-debug": "//nodejs:nodejs14_debug_amd64_debian11",
```

Comment:
I think I forgot to add node 18 to this in my other PR

Reply:
yeah that's fine, can do in a new PR

Reply:
I noticed that I'm not exactly sure where `nodejs:16` and `nodejs:latest` is getting tagged. I took a look here https://github.com/GoogleContainerTools/distroless/blob/main/BUILD#L122-L156, and NODEJS I am just more confused.
Those with `NODEJS.update` either has `-arch` suffix or `nodejs-distro` in it

Reply:
I'll see if I can understand it a bit more, then open a PR. Or if you could point me to where I should update it, I'll open a new PR for it.

Reply:
nodejs16 became multiarch, so the base nodejs:16/latest are a multiarch [manifest lists](https://docs.docker.com/engine/reference/commandline/manifest/#inspect-a-manifest-list) that is created in https://github.com/GoogleContainerTools/distroless/blob/main/cloudbuild_docker.sh#L61

And then you have to sign those special manifests here: https://github.com/GoogleContainerTools/distroless/blob/main/cloudbuild_cosign.sh#L38

Reply:
Thanks, I'll take a look at it.

Reply:
I've created a draft PR for building node18 images #1030
Waiting to see if tests for it goes through